src/rp2_common/pico_bootrom/include/pico/bootrom.h

```diff
@@ -237,6 +237,25 @@ static inline void __attribute__((noreturn)) reset_usb_boot(uint32_t usb_activit
     rom_reset_usb_boot(usb_activity_gpio_pin_mask, disable_interface_mask);
 }
 
+/*!
+ * \brief Reboot the device into BOOTSEL mode
+ * \ingroup pico_bootrom
+ *
+ * This function reboots the device into the BOOTSEL mode ('usb boot").
+ *
+ * Facilities are provided to enable an "activity light" via GPIO attached LED for the USB Mass Storage Device,
+ * and to limit the USB interfaces exposed.
```

Comment:
This chunk of doxygen is **identical** to that for `rom_reset_usb_boot`, which will probably look pretty confusing in the rendered Doxygen output? (both https://datasheets.raspberrypi.com/pico/raspberry-pi-pico-c-sdk.pdf and https://www.raspberrypi.com/documentation/pico-sdk/ )
Is the addition of active_low the only difference between `rom_reset_usb_boot` and `rom_reset_usb_boot_extra` ?

Reply:
It doesn't use a mask